### DISCUSSION TITLE
Extra ender chest rows don't delete items

### DISCUSSION THREAD

**Discussion by srnyx** [PurpurMC/Purpur#600] [__Dismissed__] [answered]
I give an extra ender chest row to people who have VIP. When they lose their VIP, I don't want them to lose all their items in the extra row. So, instead of deleting the items, just have them be inaccessible (still in the extra row, but they just can't see the extra row or access it).

**Comment by YouHaveTrouble** [comment]
That is gonna get resolved when Billy will rewrite saving data from extra rows to separate file

**Comment by encode42** [comment, marked as answer]
Follow #907 for updates on this!
